### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by solomonbbaale**
Added functionality to finish the game, there is a bug around current player state management
Added some functionality for showing messages using a different component.
Still need to get used to EsLint, its messing up the html when formatting.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,8 +6,10 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "lodash": "^4.17.19",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-native": "^0.63.2",
```
Comment (akeelm) [inline package.json:12]: Why did you add react-native?

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,50 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+  const gameWidth = 3;
+  const gameHeight = 3;
+  let [turn,setTurn]=useState(0);
```
Comment (akeelm) [inline src/App.js]: use consts and only lets when you need to

File: package.json (hunk 1 above)
Comment (solomonbbaale) [reply package.json:12]: Nice catch, hmm I think it got added when I did an npm install.
I was having some problems with using the useState hooks inside of functional components and decided to delete the whole nodule_modules folder. This might have caused that.
(Turns out the problem with using the hooks was because the functional components have to start with a capital letter, took me a while to figure that out.
Thanks will fix that.
Thanks @akeelm

File: src/App.js (hunk 2 above)
Comment (solomonbbaale) [reply src/App.js]: Do you mean this in the sense of let **[turn,setTurn]=useState(0)**; I thought since turn is always going to be changed with by the setTurn method, it should not be a constant. 
I may be totally off though. 
Will try and look up the documentation on this, am a bit confused. 
Thanks @akeelm

Comment (akeelm) [reply src/App.js]: Pretty much everywhere you've got let, you can use const.

Yeah, just read up the difference between const and let
